src/main/java/fi/dy/masa/tweakeroo/util/InventoryUtils.java

```diff
@@ -51,56 +51,64 @@
     private static final List<Integer> REPAIR_MODE_SLOT_NUMBERS = new ArrayList<>();
     private static final HashSet<Item> UNSTACKING_ITEMS = new HashSet<>();
     private static final List<Integer> TOOL_SWITCHABLE_SLOTS = new ArrayList<>();
+    private static final List<Integer> TOOL_SWITCH_IGNORED_SLOTS = new ArrayList<>();
 
     public static void setToolSwitchableSlots(String configStr)
     {
         parseSlotsFromString(configStr, TOOL_SWITCHABLE_SLOTS);
     }
 
+    public static void setToolSwitchIgnoreSlots(String configStr)
+    {
+        parseSlotsFromString(configStr, TOOL_SWITCH_IGNORED_SLOTS);
+    }
+
     public static void parseSlotsFromString(String configStr, Collection<Integer> output)
     {
         String[] parts = configStr.split(",");
         Pattern patternRange = Pattern.compile("^(?<start>[0-9])-(?<end>[0-9])$");
 
         output.clear();
 
-        for (String str : parts)
-        {
-            try
+        if (!configStr.isBlank()){
```

Comment:
I would put this as its own separate check after the clear and before the loop, and just return from the method, mainly to avoid the indentation increase of the entire code block.
Also to follow my "code style", it should be:
```java
    output.clear();

    if (configStr.isBlank())
    {
        return;
    }

    for (...
```
```